### PR TITLE
Nm quick ee instructions update

### PR DESCRIPTION
**Pull request by theguy951357**
## Describe your changes
changed instructions for netmaker ee installation and fixed a metrics exporter and prometheus bug

## Provide Issue ticket number if applicable/not in title
NET-422
## Provide testing steps
on NET-422
## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [x] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [x] If it is a bugfix, my code is <= 200 lines.
- [x] My functions are <= 80 lines.
- [x] I have had my code reviewed by a peer.
- [x] My unit tests pass locally.
- [x] Netmaker is awesome.

### CODE REVIEW

**Review by mattkasun — APPROVED** [review]

**Review by Aceix — APPROVED** [review]

File: config/config.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ type ServerConfig struct {
 	MetricsExporter            string    `yaml:"metrics_exporter"`
 	BasicAuth                  string    `yaml:"basic_auth"`
 	LicenseValue               string    `yaml:"license_value"`
-	NetmakerAccountID          string    `yaml:"netmaker_account_id"`
+	NetmakerAccountID          string    `yaml:"netmaker_tenant_id"`
```
Comment (abhishek9686) [inline config/config.go:74]: even the struct field can be changed to `NetmakerTenantID`

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: ee/license.go [hunk 2]
```diff
@@ -48,9 +48,9 @@ func AddLicenseHooks() {
 // if license is not valid, server should terminate
 func ValidateLicense() error {
 	licenseKeyValue := servercfg.GetLicenseKey()
-	netmakerAccountID := servercfg.GetNetmakerAccountID()
+	NetmakerTenantID := servercfg.GetNetmakerTenantID()
```
Comment (abhishek9686) [inline ee/license.go:51]: local variable shouldn't start with capital letter `NetmakerTenantID` -> `netmakerTenantID`

File: servercfg/serverconf.go [hunk 3]
```diff
@@ -670,13 +670,13 @@ func GetLicenseKey() string {
...
 }
 
-// GetNetmakerAccountID - get's the associated, Netmaker, account ID to verify ownership
-func GetNetmakerAccountID() string {
-	netmakerAccountID := os.Getenv("NETMAKER_ACCOUNT_ID")
-	if netmakerAccountID == "" {
-		netmakerAccountID = config.Config.Server.NetmakerAccountID
+// GetNetmakerTenantID - get's the associated, Netmaker, tenant ID to verify ownership
+func GetNetmakerTenantID() string {
+	NetmakerTenantID := os.Getenv("NETMAKER_TENANT_ID")
```
Comment (abhishek9686) [inline servercfg/serverconf.go:675]: `NetmakerTenantID` -> `netmakerTenantID`

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Comment by abhishek9686** [comment]
@gabrielseibel1  please make a note to update the version template in the next release before updating tenants 
in the version template change the `NETMAKER_ACCOUNT_ID` -> `NETMAKER_TENANT_ID`

**Pull request merged**